### PR TITLE
feat(eslint): Set up project-specific linting settings

### LINKED ISSUE #27: Project specific linter
## What is the user story?

As a developer, I want my editor to use the same linting settings as my coworkers, so that we don't work at cross-purposes when formatting code or get distracted by excessive linting on master-branch code.

## Description

* Install `eslint` and `eslint-config-fullstack` (or AirBnB, or Standard, or any other suitable config) as dev deps
* Install any needed peer deps (e.g. `eslint-plugin-react` for the `fullstack` config) as dev deps
* Create a project-specific `.eslintrc.json` which `"extends": "fullstack"` (or whatever config)
* Customize any `"rules"`
* Possibly set the `"parser": "babel-eslint"` and install `babel-eslint`, if the config doesn't do that for you – see the README of your config provider
* Commit it all to your repo
* Enjoy synchronized linting zen

### PR DESCRIPTION
**Pull request by jenktam**
See .eslintrc.json file

### Assignee Tasks

- [ ] added unit tests
- [ ] referenced any relevant issues

### Guidelines

Please add a description of this Pull Request's motivation/user story, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant info.

---

Closes #27

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "fullstack",
+  "parser": "babel-eslint",
+  "rules": {
+        "semi": ["error", "always"],
```
Comment (glebec) [inline .eslintrc.json]: You didn't enjoy your brief foray into no-semicolon style in Bones? 😛

File: package.json [hunk 2]
```diff
@@ -13,11 +13,15 @@
...
     "axios": "^0.15.3",
+    "babel-eslint": "^7.2.3",
     "babel-preset-env": "^1.5.2",
     "base64-url": "^1.3.3",
     "bluebird": "^3.5.0",
     "body-parser": "^1.16.1",
     "connect-session-sequelize": "^4.1.0",
+    "eslint": "^4.1.1",
+    "eslint-config-fullstack": "^3.0.0",
+    "eslint-plugin-react": "^7.1.0",
```
Comment (glebec) [inline package.json]: these are all dev deps, not normal deps.

File: package.json [hunk 3]
```diff
@@ -13,11 +13,15 @@
   "license": "ISC",
   "dependencies": {
     "axios": "^0.15.3",
+    "babel-eslint": "^7.2.3",
```
Comment (glebec) [inline package.json]: Don't forget this one too.

File: package.json (hunk 2 above)
Comment (jenktam) [inline package.json]: Got it!

**Pull request merged**